### PR TITLE
Feature/toastify alert install

### PR DESCRIPTION
**Pull request by bhohnco**
## Type of Change
- [ ] JSX
- [ ] HTML
- [X] JavaScript
- [X] CSS
- [ ] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

## Please describe the change made
Switched out Popup Components for React Toastify popup alerts 

## Please give a short description of the changes that were made below

- Fix:

- Feature:
* All messages have been adjusted to show the right color message depending on success or error 
* Add contacts popup has been switched out and is now shows the added contact name once submitted 
* CSS adjusted to have the popUp message fit in the middle of the screen (Mobile) 
* New message alerts tested with each user path 
* PopUp messages will need to be adjusted for different screen sizes, but I think we can adjust the NPM CSS file that came with React Toastify 

## How has this been tested?
- [X] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman
- [X] Mobile

### CODE REVIEW

File: src/Components/AddContact/AddContact.js [hunk 1]
```diff
@@ -27,9 +27,11 @@ function AddContact({ user }) {
   const [phoneNumber, setPhone] = useState('');
   // const [createContact, { loading: mutationLoading, error: mutationError, data }] = useMutation(CREATE_CONTACT);
   const [createContact, { loading: mutationLoading, error: mutationError }] = useMutation(CREATE_CONTACT);
+  const addedContactAlert = () => toast.success(`${firstName} ${lastName} contact information has been added`);
```
Comment (pcmueller) [inline src/Components/AddContact/AddContact.js:30]: So for future reference, can we just do `toast.whatever` to indicate the color/type of pop-up alert?

File: src/Components/AddContact/_addcontact.scss [hunk 2]
```diff
@@ -65,6 +65,16 @@ h2, h3 {
   display: flex;
 }
 
+@media screen and (max-width: 376px) {
+
+
+  .contact-form {
```
Comment (pcmueller) [inline src/Components/AddContact/_addcontact.scss:71]: AddContact mobile view looks much better - nice work.

File: src/Components/App/_app.scss [hunk 3]
```diff
@@ -22,6 +22,16 @@ body,
...
 }
 
+.toast-container {
+  width: 30%;
+  height: 25%;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  margin-right: -50%;
```
Comment (pcmueller) [inline src/Components/App/_app.scss:32]: Glad this worked!

**Review by pcmueller — APPROVED** [review]
Nice work Bryan, appreciate you taking on this refactor.

**Review by carolineshowscode — APPROVED** [review]
Very pleased with how this turned out, great work.

**Pull request merged**